pkg/auth/oauth/registry/tokentimeouts.go

```diff
@@ -201,15 +203,27 @@ func (a *oauthTokenTimeoutValidator) flush(flushHorizon time.Time) {
 		patch := []byte(fmt.Sprintf(timeoutsInPatch, td.token.TimeoutsIn, newTimeout))
 		_, err := a.tokens.Patch(td.token.Name, ktypes.JSONPatchType, patch)
 		if err != nil {
+			// if the token has been deleted, we should not retry
+			if !kerrors.IsNotFound(err) {
+				failedPatches = append(failedPatches, td)
```

Comment:
In what case Patch can fail in a way that will not also fail again the next time ?
IE, when is it ever worth to retry ?

Reply:
I changed this to only retry on conflict and timeout (just like `oc adm migrate`).  Retrying on conflict is fine because the validation will cause a different error if the master was trying to lower the value.  Retrying on timeout is safe because it means the server did not process your request, so no update has occurred (and thus the token may timeout incorrectly now).